yarn.lock

```diff
@@ -226,6 +226,16 @@
   dependencies:
```

Comment:
Wondering if there was a reason to update the yarn.lock in this PR?

Reply:
No, I'll remove that.